db/util_testing.go

```diff
@@ -917,3 +917,13 @@ func MoveAttachmentXattrFromGlobalToSync(t *testing.T, ctx context.Context, docI
 	_, err = dataStore.WriteWithXattrs(ctx, docID, 0, cas, value, map[string][]byte{base.SyncXattrName: newSync}, []string{base.GlobalXattrName}, opts)
 	require.NoError(t, err)
 }
+
+func RequireBackgroundManagerState(t *testing.T, ctx context.Context, mgr *BackgroundManager, expState BackgroundProcessState) {
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
+		var status BackgroundManagerStatus
+		rawStatus, err := mgr.GetStatus(ctx)
+		require.NoError(c, err)
+		require.NoError(c, base.JSONUnmarshal(rawStatus, &status))
```

Comment:
in testify, you have to use assert.NoError until https://github.com/stretchr/testify/commit/b074924938f86d417f1c9a845c7e8b0784d7f937 gets merged and we update it, otherwise it panics the test haness.